src/functions/ftp/external-poller/pollers/ftpPoller.ts

```diff
@@ -26,8 +26,8 @@ export class FtpPoller extends RemotePoller {
     await this.client.access(connectionDetails.config);
   }
 
-  async disconnect(): Promise<void> {
-    this.client.close();
+  disconnect(): Promise<void> {
+    return Promise.resolve(this.client.close());
   }
```

Comment:
this change is interesting... perhaps it wouldn't have made this change if the original code was `return this.client.close()`?